pages/resources/quests.mdx

```diff
@@ -0,0 +1,518 @@
+| game_title                                 | string        | The game's title                                                       |
+| game_publisher                             | string        | The game's publisher                                                   |
+
+### Quest User Status Object
+
+Represents the user's quest progression
+
+###### Quest User Status Structure
+
+| Field                     | Type              | Description                                                                                                                     |
+| ------------------------- | ----------------- | ------------------------------------------------------------------------------------------------------------------------------- |
+| user_id                   | snowflake         | The user's id                                                                                                                   |
+| quest_id                  | snowflake         | The quest's id                                                                                                                  |
+| enrolled_at               | ISO8601 timestamp | When the user accepted the quest                                                                                                |
+| completed_at              | ISO8601 timestamp | When the user completed the quest                                                                                               |
+| claimed_at                | ISO8601 timestamp | When the user claimed the quest's reward                                                                                        |
+| claimed_tier              | unknown           | Unused                                                                                                                          |
+| last_stream_heartbeat_ack | ISO8601 timestamp | Timestamp of the latest stream heartbeat request                                                                                |
+| stream_progress_seconds   | ISO8601 timestamp | The total amount of seconds the user has been streaming the game for since the quest started and before the quest was completed |
+| dismissed_quest_content   | ISO8601 timestamp | Whenever the user has dismissed the quest                                                                                       |
+| progress                  | object            | Unused                                                                                                                          |
+
+##### Example User Status
+
+```json
+{
+  "user_id": "222069018507345921",
+  "quest_id": "alien-quest",
+  "enrolled_at": "2077-01-01T11:59:59+00:00",
+  "completed_at": "2077-01-01T11:59:59+00:00",
+  "claimed_at": "2077-01-01T11:59:59+00:00",
+  "claimed_tier": null,
+  "last_stream_heartbeat_at": "2077-01-01T11:59:59+00:00",
+  "stream_progress_seconds": 1800,
+  "dismissed_quest_content": 0,
+  "progress": {}
```

Comment:
This appears to be a map of strings to an object with schema
```js
{
    eventName: i.event_name,
    value: i.value,
    updatedAt: i.updated_at,
    completedAt: i.completed_at
};
```

No idea what it is though

Reply:
`updatedAt` could be an early version of `last_stream_heartbeat_at`
same with, `completedAt` it could be an early version of `completed_at` in the user status object

Dunno about `event_name` and `value` though

Reply:
Resolving for now since I think that object might just be a leftover from when the feature was still called Drops